docs/debugger/just-my-code.md

```diff
@@ -6,300 +6,300 @@ ms.assetid: 0f0df097-bbaf-46ad-9ad1-ef5f40435079
+
+The JavaScript debugger classifies code as user or non-user in this order:
+
+1. The default classifications.
+   - Script executed by passing a string to the host-provided `eval` function is **MyCode**.
+   - Script executed by passing a string to the `Function` constructor is **LibraryCode**.
+   - Script in a framework reference, such as WinJS or the Azure SDK, is **LibraryCode**.
+   - Script executed by passing a string to the `setTimeout`, `setImmediate`, or `setInterval` functions is **UnrelatedCode**.
+
+2. Classifications specified for all Visual Studio JavaScript projects in the *%VSInstallDirectory%\JavaScript\JustMyCode\mycode.default.wwa.json* file.
+
+3. Classifications in the *mycode.json* file of the current project.
+
+Each classification step overrides the previous steps.
+
+All other code is classified as **MyCode**.
+
+You can modify the default classifications, and classify specific files and URLs as user or non-user code, by adding a *.json* file named *mycode.json* to the root folder of a JavaScript project. See [Customize JavaScript Just My Code](#BKMK_JS_Customize_Just_My_Code).
 
 <a name="BKMK_JS_Stepping_behavior"></a>
-During JavaScript debugging: 
+During JavaScript debugging:
 
-- If a function is non-user code, **Debug** > **Step Into** (or **F11**) behaves the same as **Debug** > **Step Over** (or **F10**).  
-- If a step begins in non-user (**LibraryCode** or **UnrelatedCode**) code, stepping temporarily behaves as if Just My Code isn't enabled. When you step back to user code, Just My Code stepping is re-enabled.  
+- If a function is non-user code, **Debug** > **Step Into** (or **F11**) behaves the same as **Debug** > **Step Over** (or **F10**).
+- If a step begins in non-user (**LibraryCode** or **UnrelatedCode**) code, stepping temporarily behaves as if Just My Code isn't enabled. When you step back to user code, Just My Code stepping is re-enabled.
 - When a user code step results in leaving the current execution context, the debugger stops at the next executed user code line. For example, if a callback executes in **LibraryCode** code, the debugger continues until the next line of user code executes.
-- **Step Out** (or **Shift**+**F11**) stops on the next line of user code. 
+- **Step Out** (or **Shift**+**F11**) stops on the next line of user code.
+
+If there's no more user code, debugging continues until it ends, hits another breakpoint, or throws an error.
 
-If there's no more user code, debugging continues until it ends, hits another breakpoint, or throws an error. 
+Breakpoints set in code are always hit, but the code is classified.
 
-Breakpoints set in code are always hit, but the code is classified.  
+- If the `debugger` keyword occurs in **LibraryCode**, the debugger always breaks.
```

Comment:
@tfosmark This showed up as a text change, but I don't see any actual change when comparing the two versions except line numbering.